### ISSUE TITLE
qemu-system-x86_64: failed to initialize kvm: Permission denied

### ISSUE THREAD

**Issue by GopherJ** [sickcodes/Docker-OSX#55]
ssh-keygen: generating new host keys: RSA DSA ECDSA ED25519 
qemu-system-x86_64: warning: dbind: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-LW022K2LIb: Connection refused
Could not access KVM kernel module: Permission denied
qemu-system-x86_64: failed to initialize kvm: Permission denied

**Comment by GopherJ** [comment]
I followed all the instructions but still it gives me this error

**Comment by sickcodes** [comment]
OS?

**Comment by GopherJ** [comment]
@sickcodes sorry it's Linux mint v19.3 cinamon

**Comment by sickcodes** [comment]
try all the troubleshooting instructions
namely, try 
`sudo chmod 666 /dev/kvm`

**Comment by GopherJ** [comment]
@sickcodes works like a charm thanks

**Comment by netsafe** [comment]
The only right way is to add the unprivileged user to kvm group, do **not** make chmod 666 ! Instead, do this as a root
`usermod -a -G kvm yourUserName`
